### PR TITLE
League Command added

### PR DESCRIPTION
**Pull request by poncethecat**
I added the league command. It's a bit rough, but it should work.

### CODE REVIEW

**Comment by poncethecat** [comment]
I just realized I also forgot to modify the config file template lol.

File: src/main/java/xyz/gnarbot/gnar/Bot.java [hunk 1]
```diff
@@ -91,6 +97,8 @@ public static MyAnimeListAPI getMALAPI() {
         return malAPI;
     }
 
+    public static RiotApi getRiotAPI() { return riotAPI; }
+
```
Comment (ArsenArsen) [inline src/main/java/xyz/gnarbot/gnar/Bot.java]: style inconsistency

File: src/main/java/xyz/gnarbot/gnar/Bot.java [hunk 2]
```diff
@@ -38,6 +40,10 @@
 
     private static final MyAnimeListAPI malAPI = new MyAnimeListAPI(KEYS.getMalUsername(), KEYS.getMalPassword());
 
+    private static final ApiConfig config = new ApiConfig().setKey(Bot.KEYS.getRiotToken());
+    private static final RiotApi riotAPI= new RiotApi(config);
+
+
```
Comment (ArsenArsen) [inline src/main/java/xyz/gnarbot/gnar/Bot.java]: too much whitespace

Comment (poncethecat) [inline src/main/java/xyz/gnarbot/gnar/Bot.java]: go die

File: src/main/java/xyz/gnarbot/gnar/Bot.java (hunk 1 above)
Comment (poncethecat) [inline src/main/java/xyz/gnarbot/gnar/Bot.java]: this i actually should fix lol

**Pull request closed without merging**